### DISCUSSION TITLE
How to load internal javascript files in nuxt.config.js

### DISCUSSION THREAD

**Discussion by bastianhilton** [nuxt/nuxt#15914] [Questions]
Hello in Nuxt 2 I was able to load internal javascript files like this:

```
script: [
      { src: '/assets/scripts/main.js', mode: 'client'},
      { src: '/assets/mdb/plugins/js/all.min.js', mode: 'client'},
      { src: '/assets/mdb/plugins/js/wysiwyg.min.js', mode: 'client'},
      { src: '/assets/mdb/js/mdb.min.js', mode: 'client'},
      { src: 'https://kit.fontawesome.com/5f36ca3a3b.js', crossorigin: 'anonymous', mode: 'client'}
    ]
```

but in nuxt 3 its not working.

I tried this but only the external file from fontawesome works:
```
app: {
    head: {
      script: [
        'assets/scripts/main.js',
        'assets/mdb/plugins/js/all.min.js',
        'assets/mdb/js/mdb.min.js',
        {
          src: 'https://kit.fontawesome.com/*****.js',
          crossorigin: 'anonymous',
          mode: 'client'
        }
      ],
    }
  },
```

Any suggestions?

**Comment by TheAlexLichter** [comment]
You'd have to put the JS files in `public` and refer to them as e.g. `/scripts/main.js` then. This should work
But ideally, you don't load that many 3rd party files and incorporate them in the build process.

**Reply by Baroshem** [reply]
Agree, this is not a good approach (especially for performance reasons) to load that many scripts in the head of the application.

If you still need it in that form, what about using `useHead` for that? -> https://nuxt.com/docs/api/composables/use-head#usehead
